samples/openapi3/client/petstore/python-pydantic-v1/tests/test_model.py

```diff
@@ -187,6 +187,28 @@ def test_anyOf_array_of_integers(self):
         except ValueError as e:
             self.assertTrue("ensure this value is less than or equal to 255" in str(e))
 
+         # test from_json, schema 3
+        json_str = '"#123456"'
+        p = petstore_api.AnyOfColor.from_json(json_str)
+        self.assertIsInstance(p.actual_instance, str)
+        self.assertEqual(p.actual_instance, '#123456')
+
+        # test to_json, schema 3
+        p = petstore_api.AnyOfColor(actual_instance='#123456')
+        self.assertEqual(p.to_json(), '"#123456"')
+
+         # test from_dict, schema 3
+        obj = '#123456'
+        p = petstore_api.AnyOfColor.from_dict(obj)
+        self.assertIsInstance(p.actual_instance, str)
+        self.assertEqual(p.actual_instance, '#123456')
+
+        # test to_dict, schema 3
+        p = petstore_api.AnyOfColor(actual_instance='#123456')
+        self.assertEqual(p.to_dict(), '#123456')
```

Comment:
Before this change, this would render as `'"#123456"'`

which is incorrect for `to_dict`